hack/ginkgo-e2e.sh

```diff
@@ -127,6 +127,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --repo-root="${KUBE_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  --network="${NETWORK:-e2e}" \
```

Comment:
yeah, that would work. 

the problem with using `NETWORK` is the flow - they're set in the cluster scripts, but the cluster scripts don't call `hack/ginkgo-e2e.sh`, so `NETWORK` won't be set here.